ansible/playbook.yaml

```
@@ -1,123 +1,95 @@
-         user: "test"
-         pass: "<password here>"
-         format:
-           time: "%H:%M:%S"
-           datetime: "%d.%m.%Y %H:%M:%S"
-         chat:
-           value: "1254c44a"
-           licht: "xwy332ou"
-           ping: "v112y1vk"
-       watchdog:
-         enable: 0
-       environment:
-         name: ""
-         device: "/dev/i2c-2"
-         data:
-           interval: 6000
-       garage:
-         enable: 0
-       pwr:
-         enable: 0
-       light:
-         timeout: 60
-       validator:
-         'test': "[14, 15, 13, 15, 11, 15, 7, 15]"
+    # As the other tasks are running with user privileges, they can not change session recording state
+    # So we run this task as root user first to disable recording
+    - name: Disable session recording
+      become: yes
+      elektra:
+        record:
+          enable: no
+
+    - name: Set keys for Opensesame
+      elektra:
+        record:
+          # Skip anything to do with session recording, as the user executing 
+          # this task lacks the privileges to activate or deactive recording
+          skip: yes
+        keys:
+          - user:/sw/libelektra/opensesame/#0/current:
+              debug:
+                backtrace: full
+                ping:
+                enable: 1
+                timeout: 24
+              audio:
+                bell: "{{ os_audio_bell | default('/dev/null', true) }}"
+                alarm: "{{ os_audio_alarm | default('/dev/null', true) }}"
+              location:
+                latitude: 48.210033
+                longitude: 16.363449
+              nextcloud:
+                url: "https://example.com/nextcloud"
+                user: "{{ os_nc_user }}"
+                pass: "{{ os_nc_pass }}"
+                format:
+                  time: "%H:%M:%S"
+                  datetime: "%d.%m.%Y %H:%M:%S"
+                chat:
+                  value: "1254c44a"
+                  licht: "xwy332ou"
+                  ping: "v112y1vk"
+              environment:
+                name: "{{ os_environment_name }}"
+                device: "{{ os_environment_device | default('/dev/i2c-2', true) }}"
+                data:
+                  interval: "{{ os_environment_data_interval | default('6000', true) }}"
+              garage:
+                enable: "{{ os_garage_enable | default('0', true) }}"
+              pwr:
+                enable: "{{ os_pwr_enable | default('0', true) }}"
+              watchdog:
+                enable: "{{ os_watchdog_enable | default('0', true) }}"
+              bell:
+                enable: "{{ os_dell_enable | default('0', true) }}"
+              light:
+                timeout: 60
+              validator:
+                'test': "[14, 15, 13, 15, 11, 15, 7, 15]"            
+ 
 - name: Configure opensesame specific to schreibtisch
   hosts: schreibtisch
   collections:
    - elektra_initiative.libelektra
   tasks:
    - name: set keys for opensesame
```

I assume it adds? (leaves the other keys unchanged)

```suggestion
   - name: add keys for opensesame
```